app/components/scroll-container.js

```diff
@@ -10,7 +10,9 @@ export default class ScrollContainerComponent extends Component {
 
   @tracked collection;
   @tracked currentItem;
+  @tracked previewing;
   @tracked itemHeight;
+  lastCurrentItem;
```

Comment:
Shouldn't this be tracked?

Reply:
It did work correctly without it.... I try it out tomorrow with tracked.

Reply:
After reviewing, its not needed.